commands/ps.js

```diff
@@ -13,13 +13,28 @@ function * run (context, heroku) {
 
   let db = yield fetcher.database(app, database)
 
+  let waiting_query = `
+SELECT EXISTS(
+  SELECT 1 FROM information_schema.columns WHERE table_schema = 'pg_catalog'
+    AND table_name = 'pg_stat_activity'
+    AND column_name = 'waiting'
+) AS available
+`
+
+  let waiting_output = yield psql.exec(db, waiting_query)
+  let waiting = 'CASE WHEN wait_event IS NULL THEN false ELSE true END AS waiting'
+
+  if (waiting_output.includes('t')) {
```

Comment:
This is error-prone because `.psqlrc` can include stuff that will send a `t` to the output, but we're making that same mistake elsewhere so I think that's okay for now.